src/Image/Cache.php

```diff
@@ -62,125 +55,115 @@ class Cache
      * @throws ImageException
      * @return array             An array with two elements: The local path to the image and the image extension
      */
-    static function resolve_url($url, $protocol, $host, $base_path, Dompdf $dompdf)
+    static function resolve_url($url, $protocol, $host, $base_path, Options $options)
```

Comment:
The doc block is still referencing the old `$dompdf` parameter. Could also remove the `@throws ImageException` while at it, as the exception is never actually thrown.